scripts/build-scenario.ts

```diff
@@ -0,0 +1,76 @@
+// Build a scenario with given ID from CLI args
+
+// Usage: `yarn scenario [ID]`
+// ID = name of scenario script OR 'default' if no ID provided
+
+import { join } from 'path'
```

Comment:
Although this is convenient I would argue that this pollutes the current namespace with a very generic name. I would prefer to import path completely or to rename `join` in order to clarify the meaning. Taken out of context I would argue that it makes it harder to read.

The same foes for `promises` below.

Reply:
How do you propose we clarify the import statement? I don't see a way to accomplish this while meeting both of these goals at the same time:
1) only importing the methods we need (like `join()` and `promises`)
2) keeping a clear name like `path.join()`

Full docs:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

Reply:
I am not sure what would be best practice in this case and I am sure you'd find a number of implementation doing this exact thing. I'd suggest using something like:
```
import { join as pathJoin } from 'path'
import { promises as fsPromises } from 'fs'
```
But if you have a good reason not to do this I'll accept the current solution as is.

Reply:
Resolved as `won't fix` after discussion.